agent/agent-configuration.mdx

```diff
@@ -10,18 +10,34 @@ and communicate with them.
 capable of and what is expected from them.
 * Using Agents is as easy as a normal communication in a chat window.
 
-## Creating an Agent
-On the [Agent](/get-started/key-concepts/agent) page, click on the "+ Create agent" button 
-located on the top right.
-This will prompt you to a page with two options:
-* Build your Agent from scratch
-* Use a template
-
-  <p align="center">
-  <img width="700" height="500" src="/images/agent/create-agent.png" />
-  </p>
-
-As the title suggests, under templates most settings options are prefilled. For this guide, we select  "Market Research Marco".
+## Create a new agent
+You can create a brand new agent, or use one of our existing [agent templates](/agents/agent-templates). Here's how to create an agent from scratch!
+
+1. Login to your [Relevance AI](https://relevanceai.com/) account ([app.relevanceai.com](https://relevanceai.com/)).
+2. Go to the [Agents](/agents) page (in the sidebar).
+3. Click "**+ New**" in the top right-hand corner of the page.
+4. Click "**+ Create Agent**" under "Blank agent" to create a new agent from scratch.
+5. Give your agent a name, description and optional profile image.
+
+(*At Relevance, we have a happy onboarding ritual where we give our agents a cute new pixel profile pic when they've passed their probation period and are ready to work with us!*)
```

Comment:
lets avoid using the word cute in this case.

Reply:
Thank you, I removed that line :)